### DISCUSSION TITLE
Plugin suggestion: Button that updates manually your properties file

### DISCUSSION THREAD

**Discussion by Steierer420** [runelite/runelite#16868] [Ideas]
Hello,

I am using two different Profiles for Plugin settings and I feel like sometimes it doesnt updates the properties-file of one profile. 

Especially I'm using the tileman plugin and it would be nice to make sure it saved my marked tiles into the properties file successfully

So my suggestion would be just a sidepanel button, that if you click it, it updates your properties and write down a message in the chatbox for example: Your properties have been saved: Tileman-122371063233400.properties

That plugin would help me alot.

**Comment by Nightfirecat** [comment]
What times have you seen your properties file not be updated? To our knowledge, that shouldn't ever be a problem.

**Comment by Steierer420** [comment]
I noticed it yesterday. I tested it with my notes: added 2 notes before I closed the clients on 2 profiles: 1 note was still there, 1 note was gone.
